### PR TITLE
Added new string in Japanese.

### PR DESCRIPTION
**Pull request by cssho**

### CODE REVIEW

File: bin/MessagesBundle_ja.properties [hunk 1]
```diff
@@ -218,31 +218,31 @@ lib_in_dir=ディレクトリ内のライブラリ:
...
-BottomMargin=Bottom margin (cm)
-LeftMargin=Left margin (cm)
-RightMargin=Right margin (cm)
-Margins_too_large=Margins are too large!
+TopMargin=上マージン (cm)
+BottomMargin=下マージン (cm)
+LeftMargin=左マージン (cm)
+RightMargin=右マージン (cm)
+Margins_too_large=マージンが大きすぎます!
 Print_outside_regions=Margins are such that the drawing will be printed in regions that are outside the printable area of your printer. Continue anyway?
```
Comment (DarwinNE) [inline bin/MessagesBundle_ja.properties]: Hi, I noticed that you did not translate this resource. Maybe the original message was not very clear. Do you need any information about it?

**Comment by DarwinNE** [comment]
Thanks @cssho !!!

File: bin/MessagesBundle_ja.properties (hunk 1 above)
Comment (cssho) [inline bin/MessagesBundle_ja.properties]: Good translation didn't think of. I'll think whether it can't be translated well a little more.

Comment (DarwinNE) [inline bin/MessagesBundle_ja.properties]: Hi @cssho 
Thanks again, I will leave open Issue #27 for that, but the translation is now much more complete, great work! 

Kind regards,
Davide

**Pull request merged**